tests/php/Chat/ChatManagerTest.php

```diff
@@ -433,4 +434,125 @@ public function testClearHistory(): void {
 			);
 		$manager->clearHistory($chat, 'users', 'admin');
 	}
+
+	public function dataSearchIsPartOfConversationNameOrAtAll(): array {
+		return [
+			['a', 'test', true],
+			['h', 'test', true],
+			['A', 'test', false],
+			['H', 'test', false],
+			['T', 'test', true],
+			['t', 'test', true],
+			['notbeginingall', 'test', false],
+			['notbegininghere', 'test', false],
+			['notbeginingtest', 'test', false]
+		];
+	}
+
+	/**
+	 * @dataProvider dataSearchIsPartOfConversationNameOrAtAll
+	 */
+	public function testSearchIsPartOfConversationNameOrAtAll(string $search, string $roomDisplayName, bool $expected): void {
+		$reflector = new \ReflectionClass('OCA\Talk\Chat\ChatManager');
+		$method = $reflector->getMethod('searchIsPartOfConversationNameOrAtAll');
+		$method->setAccessible(true);
+
+		$actual = $method->invokeArgs($this->chatManager, [$search, $roomDisplayName]);
+		$this->assertEquals($expected, $actual);
+	}
+
+	public function dataAddConversationNotify(): array {
+		return [
+			[
+				'',
+				function () {
+					$room = $this->createMock(Room::class);
+					$room->expects($this->once())
+						->method('getType')
+						->willReturn(Room::TYPE_ONE_TO_ONE);
+					return $room;
+				},
```

Comment:
Please move the `createMock` to the tests. Makes the data sets better forseeable what they try to test:
e.g. https://github.com/nextcloud/spreed/blob/acf6e50e3d6d22d8328bb831631e307c5cedbcd2/tests/php/Chat/Parser/SystemMessageTest.php#L135